<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Obsługa pliku</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>361</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Generowanie próbek negatywnych z wybranego pliku</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonNegatives">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Generuj</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonAddFile">
        <property name="text">
         <string>Otworz</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Otwieranie pliku</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>HOG i SVM</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>176</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Oblicz cechy HOG i trenuj SVM dla:</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>46</y>
       <width>351</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonColorDetector">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Próbki ręczne</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonAutoLights">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Próbki automatyczne</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Klasyfikator wstępny</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonLightsDetector">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Próbki ręczne</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="buttonAutoColor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Próbki automatyczne</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Klasyfikator szczegółowy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>260</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Testowanie</string>
    </property>
    <widget class="QPushButton" name="buttonTestSVM">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Testuj</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Krok 1. Wybierz klasyfikator</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QListWidget" name="listDetectors"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Krok 2. Wybierz zestaw próbek do przetestowania</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QListWidget" name="listImages"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
